doc/api/http2.md

```diff
@@ -3708,6 +3713,25 @@ following additional properties:
 * `type` {string} Either `'server'` or `'client'` to identify the type of
   `Http2Session`.
 
+## Note on `:authority` and `host`
+
+HTTP/2 requires requests to have either the `:authority` pseudo-header
+or the `host` header. Using `:authority` should be preferred when
+constructing an HTTP/2 request directly, and `host` should be
+preferred when converting from HTTP/1 (in proxies, for instance).
```

Comment:
Optional suggestion:

```suggestion
or the `host` header. Use `:authority` when
constructing an HTTP/2 request directly, and `host`
when converting from HTTP/1 (in proxies, for instance).
```

Reply:
Thanks for the pointers :blue_heart: I'm not sure on this one, though... the spec only says it's a "preferred form", not imperative or anything, and I'm not sure how often this preference is honored in practice... (we didn't even allow that form to begin with)

Reply:
I think I'll apply the suggestion with `Prefer` instead of `Use`